Exec Changes

tdwebster opened the pull request:
This merges several sections into the new Development Fund Grants sections which outline the process for applying to the Dev Fund. The policy is also clarified to specify exactly how the fund can be used, this is important as we become a charity.

The limits on the Development Fund are increased, while the maximum amount of bids can be has increased in line with inflation since 2006, the bounds on the fund have stayed constant. We are thus applying a similar increase to the maximum and minimum bounds. With the increased size of SSAGO Events, we believe a loss of over £1,500 is not impossible in the future and the Dev Fund should be able to handle an extreme loss. 

The Dev Fund now has allowed use for loans to events, this has been done in the past although was never allowed in policy and is now explicitly given the power to be invested, which would allow a future exec the option to grow part of the fund.

tdwebster committed:
Initial Updates

This merges several sections into the new Development Fund Grants sections which outline the process for applying to the Dev Fund. The policy is also clarified to specify exactly how the fund can be used, this is important as we become a charity.

The limits on the Development Fund are increased, while the maximum amount of bids can be has increased in line with inflation since 2006, the bounds on the fund have stayed constant. We are thus applying a similar increase to the maximum and minimum bounds. With the increased size of SSAGO Events, we believe a loss of over £1,500 is not impossible in the future and the Dev Fund should be able to handle an extreme loss. 

The Dev Fund now has allowed use for loans to events, this has been done in the past although was never allowed in policy and is now explicitly given the power to be invested, which would allow a future exec the option to grow part of the fund.

alexbanwell1 committed:
More control over dev fund bids

Ensure that usage of the dev fund is sustainable, ensure that it is being used in accordance with the aims.

JTWoodward commented:
4.2.1 should it be chair/president – unlikely to be both as separate positions.
4.4 worth linking to where you complete the application? Is it not just currently fill out a form?
4.7 stray 'if'

Is rewording to allow grants to associate members in line with the constitution? It involves spending money on people who it is beyond the aim of the organisation to help. Will the additional requirements be sufficient to satisfy the charity commission?

Should 50% of the shop surplus be going to the Dev Fund? You added it, then Alex has removed it in his changes. Obviously, if the shop needs all of it's surplus to have the base to buy stock, then it shouldn't go to the dev fund, but if it can manage with only half that would be a positive way of supporting the fund - especially since I would expect event losses to be larger than surpluses usually, and with unpredictable frequency.

4.10 is it necessary to take oversight to the full Exec, or would it be more appropriate to say evidence must be provided to the Treasurer and timescale alterations to the Exec?

Is formatting out the headings and having it all in one section a good idea for readability?

Entire section 6 seems to have been removed and probably shouldn’t have been? 6.1 I'm not sure that this is covered elsewhere but it seems like something worth saying, 6.2 is technically covered by the COI policy, but it wouldn’t hurt to make that clear, 6.3 is probably not necessary to get that outcome (Any member can make an application, COI takes out exec, referred to reps anyway) but it’s probably worth making clear that this is allowable if it should be, or isn’t if it isn’t. 6.4 is also probably worth including if the exec should be able to approve things out of meetings (there’s an argument that they shouldn’t so any reasoning and decisions are minuted, but I’d still make it clear if that’s the case.

Something I raised after Ant’s application for the minibus magnets is related to what reps can do with an application. Obviously the final decision is only a yes or no thing, but is there scope for reps to make adjustments to bids during the discussion phase – in the case the discussion was around the design of the magnets. I’m also wondering how complete the details should be on a bid, and whether that should be specified – should it be a case of the bid is approved for the idea, then you consult more generally around specific design, or should you come with a specific design, with reps allowed to make minor alterations, or not, then once approved it’s final.

Should references to the Full committee be changed yet? If I remember correctly we changed the name in the new constitution, is it worth getting a head-start on changing them in policy?

tdwebster committed:
Merge pull request #2 from alexbanwell1/patch-1

More control over dev fund bids

tdwebster commented:
4.2.1 I left these as separate as it's only one of those and I felt topographically it looked better (although maybe the better solution would only be using 'chair' in policy and just saying somewhere president is an alternative), this also ignores the clubs that have both...

4.4. It is currently a form, although that should hopefully change by the time this policy is approved as we're currently getting rid of all forms.

4.7. I'm not sure which 'if' of the three is stray... I've added some commas though to hopefully clarify what I mean

Associates - We've added 4.2.4. also on this point which we feel is sufficient to meet the requirements. We wanted to open up applications as were there an associate run event wishing to bid for things to benefit members attending and in future then this would be allowed.

Shop - Correct, there was some debate out if it was appropriate always. Instead, there's now a "The exec may put other money in" allowance so that if it is appropriate it's clear that it can be done, there'll also be other policy about the shop soon.

4.10 - We've been removing most of the references to a specific role to the exec as a whole. That way hopefully people email exec@ssago.org and if the treasurer is absent then it's clear the other exec can respond. 

formatting - no it wasn't a good idea, I'm now trying to restore a new structure with some other headings.

6. I think most of this is covered. I think parts of this go all the way back to when the exec may only meet face-to-face and now it's just the norm that exec can make decisions remotely as basically all decisions are now taken that way. Similarly, approving things out of meetings isn't as important as there's no notice required so if all the exec were free they could easily convene a meeting by video immediately and vote, then upload the minutes where a really urgent decision is needed. 

modifications - it's probably worth adding that bids can be approved subject to modification or more details can be requested. I don't think specifying a minimum level of detail is possible/wise.

Full committee - yes we should.

JTWoodward commented:
Thanks, I think I'm happy with almost all of it now then. The only thing I would still ask about the old section 6 being removed, is are you planning to allow exec members and the exec as a whole, and if so, what will be the process for that once we become a charity? I think the default at the moment would be that it's allowed since this policy states 'any member' but then the Conflict of Interest policy would kick in to prevent a decision by the exec (presumably this is a serious conflict), then it goes to reps for now, or probably the trustees once we make the change. That's probably fine (though I think I'd prefer if exec bids had to go to reps even after we're a charity), but I think it will limit future arguments in reps to include it explicitly.

Also, I did comment on Alex's changes separately, but just so it doesn't get lost I'll repeat the one relevant bit here:
I approve of the change to allow deferring applications rather than denying if there isn’t money (4.8), but it could be worth specifying what counts as low funds (as in, in this case £2,700, I think?) just for clarity, the current explanation is rather wordy and refers away for the specific numbers (it could also help to describe what's referenced so maybe:

> 2 times the maximum bid amount specified in 4.3 plus the minimum fund amount specified in 2.1

tdwebster committed:
Updates following Jack's Comments

Reformatting is introduced into section 4. to separate the different aspects of the grant.

Clarifications are provided around bids involving members of the national exec, (4.3.6-7.) the decision as to if the funds' aims are met remains at the trustee level to ensure spending is in line with these conditions. Reps is then given a final vote to approve the spending. An additional criteria is added for bids to explain their conflicts in the bid.

The amount low-on-funds is given explicitly and allows the exec to consider the current cash flow of SSAGO in differing decisions. In 2.5.

tdwebster commented:
I've now made these changes.

@alexbanwell1 do you want to check if that does what you intended with low-on-funds defferal?

alexbanwell1 commented:
> I've now made these changes.
> 
> @alexbanwell1 do you want to check if that does what you intended with low-on-funds defferal?

Not Really tbh - the reason I left it vague was to ensure that the exec were left with a bit of flexibility on defining low on funds e.g. if they thought that the money would be needed in the near future for something, and needed to keep the money in there then they could. E.g. a Covid-Witan situation. The reason I specified how I did rather than just putting a number to it was to make sure that it was updated in line with the other numbers, and that the thinking behind it was kept (2 dev fund bids worth of money above the minimum amount). That was my thinking behind it - probably should have explaned that better

alexbanwell1 commented:
Also regarding the shop - I believe I was making these changes very quickly, and just needed to get rid of it. The issue with that is generally the shop makes a 'surplus' every year... except when we buy £2.5k of neckers and £500 of blanket badges... So then yes it will probably make a 'surplus' in future years, but will not have sufficient funds to buy the next round of neckers, or whatever it needs to sell... That was the thinking.

JTWoodward commented:
Thought that might be the case with the shop, just wanted to be sure. As for 'Low-on-funds' that's what I was thinking you meant, I was just saying that it could have been explained a little clearer, and Tom seems to have gone a little overboard. I'd agree minimum + two max bids makes sense and future proofs it, but that only works if it's clear to future execs what the meaning was.

alexbanwell1 commented:
Yep that makes sense, and yeah your definition is definitely a lot clearer, so hopefully you're happy with what I've put!

tdwebster commented:
Yeah, I added in the shop part however then realised that if the exec wanted to ask the shop for surplus on a product then there's now the mechanism to ask for 'interest' on the money that's given over. Say for neckers, loaned £2500 agreed to pay back £1000/year over 3 years as neckers get sold. That then fits in a bit more tailored to how the shop works. 

I also thought you wanted a fixed figure instead of a better explanation, so that's what I went for. I'm still not sure I like the kind of fixed figure as it is now. I have branchedd an alternative way of phrasing that I think is similar but gives exec a bit more power to do a short term shut down but means if they keep doing it then there has to be a review by reps as currently theirs nothing to stop an exec deciding it's low on funds and doing nothing to fix that.

JTWoodward commented:
Yeah, reading back, that was on me - ironically, my request for a clearer explanation wasn't particularly clear.
I think the fixed figure was more supposed to be an advisory guideline for when the Dev fund might be low on funds, rather than a strict line. I'd say it's possible to keep your wording and add a 'for example, the dev fund may be declared low on funds when ...' but that depends on how much we want to continue complicating it, at this point it's starting to feel a bit excessive.

trodaway commented:
Sorry I'm a bit late to the party but just had a quick look through the proposed changes. I think I'm happy with the planned proposals but noticed that there's quite a few places where there's inconsistent names and capitalisation being used when referring to different parts of SSAGO's structure. For example, the Exec are being referred to as both the `National Exec` and the `SSAGO Exec` throughout, but I think one or the other would be clearer, or an alternative; my personal opinion is that policy should be using full titles where appropriate, so perhaps something along the lines of `Executive Committee` (I can't remember if that's what it's referred to in the new constitution) would be more appropriate? There's also still some parts being referred to as `Full Committee` instead of the `Representative Committee` naming that was said we'd change to. Also sometimes it's `board of trustees` instead of `Board of Trustees`.

alexbanwell1 commented:
Can this be closed please
